arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,553 @@
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/OperationDetail" }
+          },
+          "202": {
+            "description": "Accepted",
+            "schema": { "$ref": "#/definitions/OperationDetail" }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/locations/{location}/operationresults/{operationId}": {
+      "get": {
+        "description": "Gets completion result of the specified operation",
+        "tags": [ "OperationResults" ],
+        "operationId": "OperationResults_Get",
+        "parameters": [
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Identifier for the operation for which to get status"
+          },
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-skip-url-encoding": true,
+            "description": "Region in which the server operation was executed"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "No Content"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AnalysisServicesResourceCreateParameters": {
+      "description": "Analysis Services creation parameters",
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Location of the Analysis Services resource"
+        },
+        "sku": {
+          "$ref": "#/definitions/ResourceSkuTier",
+          "description": "Sku of the Analysis Services resource"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": { "type": "string" },
+          "description": "Key value pairs of additional properties that can ebe specified"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AnalysisServicesResourceProperties",
+          "description": "Provision request properties"
+        }
+      },
+      "required": [
+        "location",
+        "sku",
+        "tags",
+        "properties"
+      ]
+    },
+    "AnalysisServicesResourceParameters": {
+      "description": "Provision request specification",
+      "type": "object",
+      "properties": {
+        "location": {
```

Comment:
This is the body parameter for a PATCH operation. So why would there be location over here? "location" of a resource cannot be changed once it is created, correct?